translations/en.json

```diff
@@ -356,6 +356,20 @@
         "context": "pages/policy_violation.html:43:11-86"
       }
     },
+    "recovery": {
+      "start": {
+        "description": "An email will be sent with a link to reset your password.",
+        "@description": {
+          "context": "pages/recovery/start.html:27:25-60",
+          "description": "The description of the page to initiate an account recovery"
+        },
+        "heading": "Enter your email to continue",
```

Comment:
should this say email *address*?

Reply:
I just copied what's in the designs: https://www.figma.com/design/HUysJAhv6cK6p1Pc81Fxaa/Matrix-Authentication-Service-(MAS)?node-id=5354-3370&t=lkEDnlQD00OwKSmK-4

Reply:
fair enough. It seems a bit casual to me but maybe that's what we're going for.